docker-container/launch-m1.sh

```diff
@@ -0,0 +1,28 @@
+source config.sh
+set -x
+# Clean environment from before
+rm -rf docker-magento
+
+python set-magento-version.py $m1Version
+docker-compose down -v
+docker-compose up -d
+sleep 5
```

Comment:
add comment why sleep is needed (I assume you wait for demon to launch?)

Reply:
it'd be great if you can write a while loop with explicit condition instead (eg wait-for-postgres.sh file in https://docs.docker.com/compose/startup-order/)